dialogue-okhttp-client/src/test/java/com/palantir/dialogue/OkHttpChannelTest.java

```diff
@@ -32,6 +36,9 @@ Channel createChannel(URL baseUrl) {
                 .newBuilder()
                 .dispatcher(new Dispatcher(Executors.newSingleThreadExecutor()))
                 .build();
-        return OkHttpChannel.of(client, baseUrl);
+        return Channels.create(
```

Comment:
At some point I'd love to try and condense the possible entry points to make a single idiot-proof entry point for dialogue

Reply:
Agreed, I filed #340 yesterday which should cover this :-)